src/copy.c

```diff
@@ -114,12 +553,63 @@ copy_table_to_chunk_error_callback(void *arg)
 	errcontext("copying from table %s", RelationGetRelationName(scandesc->rs_rd));
 }
 
+/*
+ * Tests if there are other before insert row triggers besides the
+ * ts_insert_blocker trigger.
+ */
+static inline bool
+has_other_before_insert_row_trigger_than_ts(ResultRelInfo *resultRelInfo)
+{
+	TriggerDesc *trigdesc = resultRelInfo->ri_TrigDesc;
+	int i;
+
+	if (trigdesc == NULL)
+		return false;
+
+	if (!trigdesc->trig_insert_before_row)
+		return false;
```

Comment:
It's not covered, it means that we don't have other kinds of triggers in the tests? Probably not so important...

Reply:
The is an optimization; if no _before row insert triggers_ are present, the function can return immediately. The `ts_insert_blocker` trigger is created automatically on every hypertable. So, at least one trigger is present and the line is not covered at the moment. However, I will add a test case for this line.

Reply:
The line is now covered by a test.